commons/src/test/java/org/infinispan/commons/marshall/ProtoStreamMarshallerTest.java

```diff
@@ -55,19 +53,4 @@ private void roundtrip(Object in) throws Exception {
          assertEquals(in, out);
       }
    }
-
-   /**
-    * BaseProtoStreamMarshaller is abstract. To test it we need to extend it and make it a concrete class.
-    */
-   private BaseProtoStreamMarshaller makeInstance() {
```

Comment:
Yup, the Base**** was just begging to be extended. Not much use without that. My only complain is the duplicate name which usually leads to confusion.

Reply:
If we can't change the `ProtoStreamMarshaller` name for compatibility reasons, at the very least I could change this test name to `ClientProtoStreamMarshallerTest`.